firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestoreSettings.java

```diff
@@ -145,6 +145,22 @@ public Builder setCacheSizeBytes(long value) {
       return this;
     }
 
+    public String getHost() {
+      return host;
+    }
+
+    public boolean isSslEnabled() {
+      return sslEnabled;
+    }
+
+    public boolean isPersistenceEnabled() {
+      return persistenceEnabled;
+    }
+
+    public long getCacheSizeBytes() {
+      return cacheSizeBytes;
+    }
```

Comment:
Please, add comments for this methods, specially the default values (because they return primitive types, it cannot be null).